src/Networking/v2/Models/LoadbalancerStat.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\OperatorResource;
+
+/**
+ * Represents Neutron v2 Loadbalancer Stats
+ *
+ * @property Api $api
+ */
+class LoadbalancerStat extends OperatorResource implements Retrievable
+{
+    /**
+     * @var integer
+     */
+    public $bytesIn;
+
+    /**
+     * @var integer
```

Comment:
@laszlof Coming up with our own type `BigInt` is just too much of reinvent the wheel here. Doctrine also employs `string` for [BigInt](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#bigint).

Secondly, `LoadBalancerStat` only implements `Retrievable` interface which means SDK users cannot mistakenly assign non-numerical character to these fields.